src/components/Footer/Footer.js

```diff
@@ -37,9 +37,8 @@ const Footer = ({
   disableComment = false,
   blockStatus,
   loadingVote,
-  showScoreButton = false,
-  onPressScore,
-  isShowDM = false
+  isShowDM = false,
+  isShortText = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isShortText` is introduced but the prop `onPressScore` and `showScoreButton` are removed. Ensure that this change doesn't affect other parts of the code where `Footer` component is used.
```diff
-  showScoreButton = false,
-  onPressScore,
+  isShortText = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->